sites/platform/src/dedicated-environments/overview.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Overview"
+weight: -20
+sidebarTitle: "Overview"
+layout: single
+description:  "Our Dedicated environments are well-suited for those who need more resources and redundancy for their production environment, along with stricter isolation requirements."
+---
+
+{{% description %}}
+
+When you create a project on Platform, you can choose to deploy it using one of three types of architecture offerings: Grid, [Dedicated Generation 2](/dedicated-environments/dedicated-gen-2) (DG2) or [Dedicated Generation 3](/dedicated-environments/dedicated-gen-3) (DG3). 
+
+## What is Dedicated?
+
+DG2 and DG3 are classified as Dedicated environments (Dedicated). This is because your production environment is replicated across at least three virtual servers that are dedicated solely to your project.
+
+In the diagram below, we can see that the Dedicated architecture provides three virtual servers that act as isolated hosts for a site in a failover configuration. Within each server, all the data of your site is synced. 
+
+![The dedicated architecture](/images/dedicated/dedicated-architecture.svg "0.50")
+
+Having three isolated hosts means that when one becomes unavailable, the others take over, so your site will always remain up and running. This differs from the Grid architecture, where a single host runs multiple projects from various customers simultaneously. 
+
+From the Grid architecture diagram below, we can see that projects hosted in Grid share resources. The CPU, memory, and networking with other projects are all running on the same host. There is only one host that can be relied on in a failover configuration, and this host also holds the resources of various other sites. There is also a Ceph storage dependency.
+
+![The grid architecture](/images/dedicated/grid-architecture.svg "0.50")
+
+## Deployment
+
+With a Dedicated environment, you are given the freedom to deploy into **any region of supported IaaS providers** (currently **AWS, Azure, GCP, OVH**). This differs from the Grid architecture, which is solely available in [public regions](https://platform.sh/regions/). 
```

Comment:
here we _DO_ refer to the IaaS providers as "IaaS providers" and not "regions" which is different from the DG3 development page. We'll need to make sure these two sections are consistent.